pyserini/search/lucene/tprob/__main__.py

```diff
@@ -0,0 +1,140 @@
+#
+# Pyserini: Reproducible IR research with sparse and dense representations
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import argparse
+import subprocess
+import sys
+import json
+import os
+sys.path.insert(0, './')
```

Comment:
I don't think this is needed anymore.

Reply:
Sorry, we didn't finish all code refactoring when sending a PR. They are all fixed now.